src/ctia/bundle/schemas.clj

```diff
@@ -43,8 +43,8 @@
 (s/defschema BundleExportOptions
   (st/optional-keys
    {:related_to [(s/enum :source_ref :target_ref)]
-    :source_type EntityTypes
-    :target_type EntityTypes
+    :source_type [EntityTypes]
+    :target_type [EntityTypes]
```

Comment:
Breaking change?

Reply:
I just read the QA instructions, seems like the routes just do the right thing.

Reply:
Yeah the way to pass list in query params is a bit counter intuitive but at least it's fully backward compatible